spec/requests/api/v1/participant_declarations_spec.rb

```diff
@@ -375,6 +375,48 @@ def build_params(attributes)
           expect(fake_logger).to have_received(:info).with(instance_of(Array)).ordered
         end
       end
+
+      context "when existing declaration" do
+        %i[awaiting_clawback clawed_back].each do |claw_state|
+          context "has state #{claw_state}" do
+            let!(:existing_participant_declaration) do
+              create(
+                :participant_declaration,
+                claw_state,
+                user: ect_profile.user,
+                cpd_lead_provider:,
+                participant_profile: ect_profile,
+                course_identifier: valid_params[:course_identifier],
+              )
+            end
+
+            it "returns 200" do
+              params = build_params(valid_params)
+              post "/api/v1/participant-declarations", params: params
+              expect(response.status).to eq 200
```

Comment:
@mooktakim here you want to assert that the a new declaration has been created and that it also is in the correct state alongside with a statement line item in the correct test.